### DISCUSSION TITLE
Help wanted: My Windows keep maximizing themselves when I enter them

### DISCUSSION THREAD

**Discussion by b-huessner** [LazyVim/LazyVim#3484] [Q&A] [answered]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.10.0

### Operating system/version

Manjaro Linux (Arch)

### Describe the bug

Hi everyone,

This may not be a Bug, and I dont know how to reproduce it, I dont know what I did to cause it. But maybe its just a feature that I accidentally activated:

As of this morning my Windows (like Splits etc.) keep maximizing whenever I enter them.
I tried the "Toggle Maximize" Keybind <leader>m or <leader>wm, but that doesnt seem to change anything.

Do you know how to turn this off? Sorry for not supplying a reproduction, I just dont know what happened/what I did.

### Steps To Reproduce

Dont know how to reproduce.

### Expected Behavior

I would have assumed, that the behavior I am witnessing is what should happen with the Maximize Toggle, but I would expect to be able to turn it back off

**Comment by ditsuke** [comment]
Here to +1 on this. I am currently trying to find the plugin responsible

**Reply by ditsuke** [reply]
If you use persistence.nvim for sessions, start anew // as @dpetka2001 suggested purge all your sessions.

**Comment by dpetka2001** [comment, marked as answer]
Delete all the files under `~/.local/state/nvim/sessions/` directory and let Neovim recreate them. With the newest LazyVim update this should not happen any more.

**Reply by b-huessner** [reply]
Okay, thank you. That has helped for now.
